patterns/1-initial/governance-levels.md

```diff
@@ -0,0 +1,76 @@
+## Title
+
+Transparent governance levels
+
+## Context
+
+Several teams are using InnerSource patterns and best practices. However the
+degree to which they welcome not only contributions but give equal collaboration
+rights to contributors differ. As a result there are unmatched expectations,
+confusion and frustration when teams collaborate across team boundaries.
+
+## Problem
+
+For two projects InnerSource best practices have been adopted. Project A
+has a shared ownership model with Trusted Committers from multiple teams.
+Project B is fully owned by one team, only contributions are coming from
+multiple teams. New contributors to either project are regularly confused about
+the level of influence they can gain to the respective project. This leads to
+long discussions, escalations and time lost on clarifications.
+
+## Forces
+
+- For project A shared ownership works well, members coming from multiple teams
+  are working together.
+- For project B the backing team would like to retain a certain level of
+  ownership and control. Sharing ownership with other Trusted Committers outside
+  of the original team is not an option.
+- Contributors want clarity on the level of influence they can gain in an
+  InnerSource project they are involved with.
+- Writing detailed guidelines into each contributions file leads to a lot of
+  text that is hard to understand for engineers.
+
+
+## Solution
+
+Establish standardised building blocks which can be used by projects to signify
+how much influence they are willing to share. Those building blocks can then be
+used in contributing files.
+
+Examples of building blocks:
+
+* Bug reports and issues welcome: People outside the core development team are
+  welcome to read the code. They can submit feature requests and bug reports for
+  things they would like to see changed.
+
+* Contributions welcome: People outside the core development team may use the
```

Comment:
Is "core development team" here the same as "backing team" which you used above?

Reply:
I think I'm lacking a good term here: What I'm trying to explain is that there is a difference between a project that is carried forward by individuals working under one line manager, delivering one roadmap vs. a project where individuals from multiple such teams collaborate.

Reply:
I see. I don't know the best term here either.
For the time being I would then suggest to just introduce and define **a term**, likely towards the beginning of the Pattern, and then stick to that term through out the pattern.

As we improve this pattern over time we might come up with another term, possibly influenced by other patterns as well. We may even want to pull such terms that we attribute certain meaning to into a central glossary to help pattern authors and readers to use these terms in a similar manner.